{"content": {"suites": {"0": {"status": {"total_pass": 1, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 1, "total_error": 0}, "tests": {"0": {"status": "FAIL", "message": "               pytest_bdd.exceptions.StepDefinitionNotFoundError: Step definition is not found: When \"client makes a http GET request to cache for patient 9730676399 on path %2FFHIR%2FR4%2FPatient and endpoint\". Line 7 in scenario \"when no PDS response cache available\" in the feature \"/Users/richa.jha/Desktop/NHS_Proxy/proxy-validated-relationships-service/tests/test_api/features/PDSCache/1_pds_cache.feature\"\n", "test_name": "test_when_no_pds_response_cache_available[%2FFHIR%2FR4%2FPatient-9730676399-]", "rerun": "0"}, "1": {"status": "PASS", "message": "", "test_name": "test_when_no_pds_response_cache_available[%2FFHIR%2FR4%2FPatient-9730676399-RelatedPerson]", "rerun": "0"}}, "suite_name": "test_api/test_pds_cache.py"}}}, "date": "July 17, 2024", "start_time": 1721220111.0296, "total_suite": 1, "status": "FAIL", "status_list": {"pass": "1", "fail": "1", "skip": "0", "error": "0", "xpass": "0", "xfail": "0", "rerun": "0"}, "total_tests": "2"}